### ISSUE TITLE
Unhandled exception executing command 'invsee playername' in invsee:

### ISSUE THREAD

**Issue by teracube** [BlockHorizons/InvSee#2]
InvSee_dev-4.phar
PocketMine-MP.phar #1216

```
2018-06-28 [11:05:52] [Server thread/CRITICAL]: Unhandled exception executing command 'invsee brice' in invsee: Call to undefined method BlockHorizons\InvSee\commands\InvSeeCommand::sendPermissionMessage()
2018-06-28 [11:05:52] [Server thread/CRITICAL]: Error: "Call to undefined method BlockHorizons\InvSee\commands\InvSeeCommand::sendPermissionMessage()" (EXCEPTION) in "InvSee_dev-4.phar/src/BlockHorizons/InvSee/commands/BaseCommand" at line 109
2018-06-28 [11:05:52] [Server thread/DEBUG]: #0 src/pocketmine/command/SimpleCommandMap(258): BlockHorizons\InvSee\commands\BaseCommand->execute(pocketmine\Player object, string invsee, array Array())
2018-06-28 [11:05:52] [Server thread/DEBUG]: #1 src/pocketmine/Server(1955): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string invsee brice)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #2 src/pocketmine/Player(2190): pocketmine\Server->dispatchCommand(pocketmine\Player object, string invsee brice)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #3 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(222): pocketmine\Player->chat(string /invsee brice)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #4 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #8 src/pocketmine/Player(3036): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #9 src/pocketmine/network/mcpe/RakLibInterface(158): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #10 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 109.30.100.6 59695, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/RakLibInterface(103): raklib\server\ServerHandler->handlePacket()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #12 src/pocketmine/network/Network(94): pocketmine\network\mcpe\RakLibInterface->process()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #13 src/pocketmine/network/mcpe/RakLibInterface(80): pocketmine\network\Network->processInterface(pocketmine\network\mcpe\RakLibInterface object)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #14 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #15 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #16 src/pocketmine/Server(2258): pocketmine\snooze\SleeperHandler->sleepUntil(double 1530176752.3676)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #17 src/pocketmine/Server(2129): pocketmine\Server->tickProcessor()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #18 src/pocketmine/Server(1710): pocketmine\Server->start()
2018-06-28 [11:05:52] [Server thread/DEBUG]: #19 src/pocketmine/PocketMine(237): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/myservertest/, string /home/myservertest/plugins/)
2018-06-28 [11:05:52] [Server thread/DEBUG]: #20 /home/myservertest/PocketMine-MP.phar(1): require(string phar:///home/myservertest/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by teracube** [comment]
@Muqsit 

log is spammed with this error
same issue or not ?
thanks

InvSee_dev-4.phar
PocketMine-MP.phar #1419 v1.6

```
2018-09-03 [16:50:39] [Server thread/CRITICAL]: Unhandled exception executing command 'invsee assoidi37' in invsee: Tried to schedule task to disabled scheduler
2018-09-03 [16:50:39] [Server thread/CRITICAL]: InvalidStateException: "Tried to schedule task to disabled scheduler" (EXCEPTION) in "src/pocketmine/scheduler/TaskScheduler" at line 150
2018-09-03 [16:50:39] [Server thread/DEBUG]: #0 src/pocketmine/scheduler/TaskScheduler(80): pocketmine\scheduler\TaskScheduler->addTask(BlockHorizons\InvSee\libs\muqsit\invmenu\inventories\tasks\DoubleChestDelayTask object, integer 4, integer -1)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #1 InvSee_dev-4.phar/src/BlockHorizons/InvSee/libs/muqsit/invmenu/inventories/DoubleChestInventory(61): pocketmine\scheduler\TaskScheduler->scheduleDelayedTask(BlockHorizons\InvSee\libs\muqsit\invmenu\inventories\tasks\DoubleChestDelayTask object, integer 4)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #2 src/pocketmine/inventory/BaseInventory(405): BlockHorizons\InvSee\libs\muqsit\invmenu\inventories\DoubleChestInventory->onOpen(pocketmine\Player object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #3 src/pocketmine/Player(3817): pocketmine\inventory\BaseInventory->open(pocketmine\Player object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #4 InvSee_dev-4.phar/src/BlockHorizons/InvSee/libs/muqsit/invmenu/InvMenu(175): pocketmine\Player->addWindow(BlockHorizons\InvSee\libs\muqsit\invmenu\inventories\DoubleChestInventory object, NULL )
2018-09-03 [16:50:39] [Server thread/DEBUG]: #5 InvSee_dev-4.phar/src/BlockHorizons/InvSee/InventoryHandler(256): BlockHorizons\InvSee\libs\muqsit\invmenu\InvMenu->send(pocketmine\Player object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #6 InvSee_dev-4.phar/src/BlockHorizons/InvSee/commands/InvSeeCommand(18): BlockHorizons\InvSee\InventoryHandler->viewInventory(pocketmine\Player object, string assoidi37)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #7 InvSee_dev-4.phar/src/BlockHorizons/InvSee/commands/BaseCommand(113): BlockHorizons\InvSee\commands\InvSeeCommand->onCommand(pocketmine\Player object, string invsee, array Array())
2018-09-03 [16:50:39] [Server thread/DEBUG]: #8 src/pocketmine/command/SimpleCommandMap(258): BlockHorizons\InvSee\commands\BaseCommand->execute(pocketmine\Player object, string invsee, array Array())
2018-09-03 [16:50:39] [Server thread/DEBUG]: #9 src/pocketmine/Server(1955): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string invsee assoidi37)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #10 src/pocketmine/Player(2191): pocketmine\Server->dispatchCommand(pocketmine\Player object, string invsee assoidi37)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(222): pocketmine\Player->chat(string /invsee assoidi37)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #13 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #14 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #15 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #16 src/pocketmine/Player(3056): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #17 src/pocketmine/network/mcpe/RakLibInterface(162): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #18 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 83.200.81.89 53074, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #19 src/pocketmine/network/mcpe/RakLibInterface(103): raklib\server\ServerHandler->handlePacket()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #20 src/pocketmine/network/Network(94): pocketmine\network\mcpe\RakLibInterface->process()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #21 src/pocketmine/network/mcpe/RakLibInterface(80): pocketmine\network\Network->processInterface(pocketmine\network\mcpe\RakLibInterface object)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #22 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #23 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #24 src/pocketmine/Server(2259): pocketmine\snooze\SleeperHandler->sleepUntil(double 1535986239.3392)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #25 src/pocketmine/Server(2134): pocketmine\Server->tickProcessor()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #26 src/pocketmine/Server(1700): pocketmine\Server->start()
2018-09-03 [16:50:39] [Server thread/DEBUG]: #27 src/pocketmine/PocketMine(243): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/myserver/, string /home/myserver/plugins/)
2018-09-03 [16:50:39] [Server thread/DEBUG]: #28 /home/myserver/PocketMine-MP.phar(1): require(string phar:///home/myserver/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by lukeeey** [comment]
That's a completely different error

**Comment by teracube** [comment]
@lukeeey 
ok i have opened issue #8

**Comment by teracube** [comment]
same issue today with latest build

```
2018-10-09 [17:35:16] [Server thread/CRITICAL]: Unhandled exception executing command 'enderinvsee ' in enderinvsee: Call to undefined method BlockHorizons\InvSee\commands\EnderInvSeeCommand::sendPermissionMessage()
2018-10-09 [17:35:16] [Server thread/CRITICAL]: Error: "Call to undefined method BlockHorizons\InvSee\commands\EnderInvSeeCommand::sendPermissionMessage()" (EXCEPTION) in "InvSee_dev-4.phar/src/BlockHorizons/InvSee/commands/BaseCommand" at line 109
2018-10-09 [17:35:16] [Server thread/DEBUG]: #0 src/pocketmine/command/SimpleCommandMap(258): BlockHorizons\InvSee\commands\BaseCommand->execute(pocketmine\Player object, string enderinvsee, array Array())
2018-10-09 [17:35:16] [Server thread/DEBUG]: #1 src/pocketmine/Server(1955): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string enderinvsee )
2018-10-09 [17:35:16] [Server thread/DEBUG]: #2 src/pocketmine/Player(2191): pocketmine\Server->dispatchCommand(pocketmine\Player object, string enderinvsee )
2018-10-09 [17:35:16] [Server thread/DEBUG]: #3 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(222): pocketmine\Player->chat(string /enderinvsee )
2018-10-09 [17:35:16] [Server thread/DEBUG]: #4 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #8 src/pocketmine/Player(3056): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #9 src/pocketmine/network/mcpe/RakLibInterface(161): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #10 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 81.242.141.12 51413, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/RakLibInterface(102): raklib\server\ServerHandler->handlePacket()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #12 src/pocketmine/network/Network(94): pocketmine\network\mcpe\RakLibInterface->process()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #13 src/pocketmine/network/mcpe/RakLibInterface(92): pocketmine\network\Network->processInterface(pocketmine\network\mcpe\RakLibInterface object)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #14 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #15 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #16 src/pocketmine/Server(2259): pocketmine\snooze\SleeperHandler->sleepUntil(double 1539099316.7081)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #17 src/pocketmine/Server(2134): pocketmine\Server->tickProcessor()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #18 src/pocketmine/Server(1700): pocketmine\Server->start()
2018-10-09 [17:35:16] [Server thread/DEBUG]: #19 src/pocketmine/PocketMine(249): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/myserver/, string /home/myserver/plugins/)
2018-10-09 [17:35:16] [Server thread/DEBUG]: #20 /home/myserver/PocketMine-MP.phar(1): require(string phar:///home/myserver/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by Muqsit** [comment]
Solved in v1.0
